package.json

```diff
@@ -33,11 +33,5 @@
     "lodash": "^4.6.1",
     "testarmada-logger": "^1.1.1"
   },
-  "devDependencies": {
-    "chai": "^3.5.0",
-    "chai-as-promised": "^6.0.0",
-    "codecov": "^3.6.5",
-    "istanbul": "^0.4.5",
-    "mocha": "^3.2.0"
```

Comment:
Damn the repo doesn't have any tests :/

Reply:
Yeah, that is par for the course man. A lot of these repos are just unnecessary, too. For instance, why have a specific "midway-logger" repo/module if it is a logger specifically used in the midway project?